ubuntu-advantage

```diff
@@ -187,6 +187,31 @@ check_user() {
 }
 
 
+print_status() {
+    local livepatch_status livepatch_output
+    if is_livepatch_enabled; then
+        livepatch_status="enabled"
+        livepatch_output=$(canonical-livepatch status)
+    else
+        livepatch_status="disabled"
+        is_supported_series "$LIVEPATCH_SUPPORTED_SERIES" || livepatch_status="$livepatch_status (not available)"
+    fi
+    echo "livepatch: $livepatch_status"
+    if [ "$livepatch_output" ]; then
+        echo "$livepatch_output"
+    fi
+
```

Comment:
Please add an extra blank line in this output between the statuses of both services so that we get a clear separation between them, or else, at first glance, when livepatch is enabled for example, the "esm:" line might look as being another item of the livepatch status:
```
ubuntu@15-89:~$ ubuntu-advantage status
livepatch: enabled
kernel: 4.4.0-89.112-generic
fully-patched: false
version: "28.1"
esm: disabled (not available)
```
+1